k4MarlinWrapper/k4MarlinWrapper/converters/EDM4hep2Lcio.h

```diff
@@ -31,6 +33,7 @@
 #include <vector>
 
 class PodioDataSvc;
+class MetadataSvc;
```

Comment:
I don't see this `MetadataSvc` name used in the header file?

Reply:
Changed to `IMetadataSvc`, which was the original intention and is used below